src/style.css

```diff
@@ -1 +1,32 @@
-/* add styles here */
+* {
+  margin: 0;
+}
```

Comment:
Don't use the `*` selector (it impacts performance). Set styles only for elements that require them. Zeroing out your margins, paddings or other styles with '*' is still inefficient for the browser.